app/src/main/java/com/framgia/forder/data/model/Product.java

```diff
@@ -0,0 +1,189 @@
+    @Expose
+    @SerializedName("shop")
+    private Shop mShop;
+    @Expose
+    @SerializedName("shop_id")
+    private int mShopId;
+
+    public Product(int id, String name, double price, String description,
+            CollectionImage collectionImage, String startHour, String endHour, String status,
+            Shop shop, int shopId) {
+        mId = id;
+        mName = name;
+        mPrice = price;
+        mDescription = description;
+        mCollectionImage = collectionImage;
+        mStartHour = startHour;
+        mEndHour = endHour;
+        mStatus = status;
+        mShop = shop;
+        mShopId = shopId;
+    }
+
+    protected Product(Parcel in) {
+        mId = in.readInt();
+        mName = in.readString();
+        mPrice = in.readDouble();
+        mDescription = in.readString();
+        mStartHour = in.readString();
+        mEndHour = in.readString();
+        mStatus = in.readString();
+        mShopId = in.readInt();
+    }
+
+    public int getShopId() {
+        return mShopId;
+    }
+
+    public void setShopId(int shopId) {
+        mShopId = shopId;
+    }
+
+    public int getId() {
+        return mId;
+    }
+
+    public void setId(int id) {
+        mId = id;
+    }
+
+    public String getName() {
+        return mName;
+    }
+
+    public void setName(String name) {
+        mName = name;
+    }
+
+    public double getPrice() {
+        return mPrice;
+    }
+
+    public void setPrice(double price) {
+        mPrice = price;
+    }
+
+    public String getDescription() {
+        return mDescription;
+    }
+
+    public void setDescription(String description) {
+        mDescription = description;
+    }
+
+    public CollectionImage getCollectionImage() {
+        return mCollectionImage;
+    }
+
+    public void setCollectionImage(CollectionImage collectionImage) {
+        mCollectionImage = collectionImage;
+    }
+
+    public String getEndHour() {
+        return mEndHour;
+    }
+
+    public void setEndHour(String endHour) {
+        mEndHour = endHour;
+    }
+
+    public String getStartHour() {
+        return mStartHour;
+    }
+
+    public void setStartHour(String startHour) {
+        mStartHour = startHour;
+    }
+
+    public Shop getShop() {
+        return mShop;
+    }
+
+    public void setShop(Shop shop) {
+        mShop = shop;
+    }
+
+    public String getStatus() {
+        return mStatus;
+    }
+
+    public void setStatus(String status) {
+        mStatus = status;
+    }
+
+    public String getFormatStartHour() {
+        return Utils.DateTimeUntils.convertUiFormatToDataFormat(mStartHour, Utils.INPUT_TIME_FORMAT,
+                Utils.OUTPUT_TIME_FORMAT);
+    }
+
+    public String getFormatEndHour() {
+        return Utils.DateTimeUntils.convertUiFormatToDataFormat(mEndHour, Utils.INPUT_TIME_FORMAT,
+                Utils.OUTPUT_TIME_FORMAT);
+    }
+
+    public String getFormatPrice() {
+        return String.format(Utils.FORMAT_PRICE, mPrice);
```

Comment:
WARNING: [DefaultLocale] Implicitly using the default locale is a common source of bugs: Use `String.format(Locale, ...)` instead
 Calling `String#toLowerCase()` or `#toUpperCase()` *without specifying an explicit locale* is a common source of bugs. The reason for that is that those methods will use the current locale on the user's device, and even though the code appears to work correctly when you are developing the app, it will fail in some locales. For example, in the Turkish locale, the uppercase replacement for `i` is *not* `I`.

If you want the methods to just perform ASCII replacement, for example to convert an enum name, call `String#toUpperCase(Locale.US)` instead. If you really want to use the current locale, call `String#toUpperCase(Locale.getDefault())` instead.